packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -174,29 +225,27 @@ export class PersonalizeMiddleware extends MiddlewareBase {
       return response;
     }
 
-    if (!browserId) {
-      browserId = await this.cdpService.generateBrowserId();
-
-      if (!browserId) {
-        debug.personalize('skipped (browser id generation failed)');
-        return response;
-      }
-    }
-
-    // Execute targeted experience in CDP
-    const { ua } = userAgent(req);
     const params = this.getExperienceParams(req);
-    const pointOfSale = this.config.getPointOfSale
-      ? this.config.getPointOfSale(site, language)
-      : PosResolver.resolve(site, language);
-    const variantId = await this.cdpService.executeExperience(
+    const { ua } = userAgent(req);
+
+    debug.personalize(
+      'executing experience for %s %s %s %o',
       personalizeInfo.contentId,
-      browserId,
       ua,
```

Comment:
Can you confirm that the Engage SDK sends the User-Agent header to CDP callflows request? I see we are only using `ua` here for logging. Previously, this was sent to our `CdpService` and added as a header on CDP request.

Reply:
Confirmed, we have UA headers in the CDP callflows request.
Will remove our UA.